src/main/java/controllers/UserManager.java

```diff
@@ -1,13 +1,48 @@
 package controllers;
 //package useCases;
 
+
+import entities.Reading;
+import entities.User;
+import useCases.LogIn;
+import useCases.ReadingLogger;
+import useCases.UserGenerator;
+
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.io.ObjectOutputStream;
+
 public class UserManager {
 
-    public static void create_user() {
-        //logs.put(user, user.getReadingLog());
+    public static void create_user(String username, String password, int year, int month, int day) {
+        UserGenerator.generateUser(username, password, year, month, day);
+    }
+
+    public static void login_user(String username, String password) {
+        try {
+            LogIn.login(username, password);
+        }
+        catch (Exception e) {
+            System.out.println("Wrong password");
```

Comment:
Should we remove the exception for login and rather use validation from handling password checks? example boolean method

Reply:
We decided to keep the exception, so I am merging